src/redux/configure-store.js

```diff
@@ -1,32 +1,24 @@
 import {compose, createStore, applyMiddleware, combineReducers} from 'redux'
-import loggerMiddleware from 'redux-logger'
-import {createHistory, createHashHistory} from 'history'
+import {syncHistory, routeReducer} from 'react-router-redux'
+import {browserHistory} from 'react-router'
 import {reduxReactRouter, routerStateReducer} from 'redux-router'
 import {apiMiddleware, authMiddleware, likesLogicMiddleware, userPhotoLogicMiddleware, redirectionMiddleware, scrollMiddleware, realtimeMiddleware} from './middlewares'
 import * as reducers from './reducers'
 
 //order matters — we need to stop unauthed async fetching before request, see authMiddleware
-let middleware = [ authMiddleware, apiMiddleware, likesLogicMiddleware, userPhotoLogicMiddleware, redirectionMiddleware, scrollMiddleware, realtimeMiddleware ]
+const middleware = [ authMiddleware, apiMiddleware, likesLogicMiddleware, userPhotoLogicMiddleware, redirectionMiddleware, scrollMiddleware, realtimeMiddleware, syncHistory(browserHistory) ]
 
-const isDevelopment = process.env.NODE_ENV != 'production'
-
-//tells webpack to include logging middleware in dev mode
-if (isDevelopment) {
-  middleware.push(loggerMiddleware())
-}
-
-let enhancers = [applyMiddleware(...middleware),
-  reduxReactRouter({ createHistory }),]
+let enhancers = [applyMiddleware(...middleware),]
 
 //tells webpack to include devtool enhancer in dev mode
-if (isDevelopment && window.devToolsExtension) {
+if (window.devToolsExtension) {
```

Comment:
Why are we removing `isDevelopment` here? It doesn't look like the part of the "router updates".